community/kernel/src/main/java/org/neo4j/kernel/impl/core/NodeProxy.java

```diff
@@ -377,6 +382,85 @@ public Object getProperty( String key, Object defaultValue )
     }
 
     @Override
+    public Map<String, Object> getProperties( String... keys )
```

Comment:
@burqen no, we use the word "keys" in the API (e.g. getPropertyKeys), so the correct word is indeed keys